documentation/apis/administration/1.0.0/api.yaml

```diff
@@ -390,7 +390,55 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/errorResponse"
-             
+
+  /services/{service}/projections/{projection}:
+    get:
+      tags:
+      - "Gestion des services"
+      - "Projection"
+      summary: "Récupération d'une projection supportée par un service."
+      description: |
+        Cette requête retourne une projection supportée par un service.
+      operationId: "get-service-projection"
+      parameters:
+      - name: "service"
+        description: "Id du service concerné"
+        in: "path"
+        required: true
+        schema:
+          type: "string"      
+      - name: "projection"
+        description: "Id de la projection concernée"
+        in: "path"
+        required: true
+        schema:
+          type: "string"
+      responses:
+        200:
+          description: "successful operation"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/projectionConfiguration"
+        400:
```

Comment:
You can delete this 400 and let the 404. I've created an issue about that. #68

Reply:
I kept the 400 since it can happen with empty service or projection. I added a test.